app/src/hooks/usePersistentState.js

```diff
@@ -1,6 +1,6 @@
 import { useState } from "react";
 
-export function userPersistentState(storageKey: string, stateObject: any) {
+export function userPersistentState(storageKey, stateObject) {
```

Comment:
why have you removed the types?

Reply:
As shared in the description of this PR:

- Turned the `userPersistentState` to a `js` file instead of a `ts` file. This was the only file in the frontend written in `ts` which was a bit incosistent.

Reply:
ah ok, i see. Carry on then.